config/event-import-handlers.php

```diff
@@ -3,19 +3,26 @@
 use App\Enums\EventServices;
 use HackGreenville\EventImporter\Services\EventBriteHandler;
 use HackGreenville\EventImporter\Services\LumaHandler;
+use HackGreenville\EventImporter\Services\MeetupGraphqlHandler;
 use HackGreenville\EventImporter\Services\MeetupRestHandler;
 
 return [
     'max_days_in_past' => env('EVENT_IMPORTER_MAX_DAYS_IN_PAST', 30),
     'max_days_in_future' => env('EVENT_IMPORTER_MAX_DAYS_IN_FUTURE', 180),
+    'meetup_graphql_private_key_path' => env('EVENT_IMPORTER_MEETUP_GRAPHQL_PRIVATE_KEY_PATH', __DIR__ . '/../app-modules/event-importer/src/Providers/Keys/meetup-graphql.pem'),
```

Comment:
```suggestion
    'meetup_graphql_private_key_path' => env('EVENT_IMPORTER_MEETUP_GRAPHQL_PRIVATE_KEY_PATH'),
```

We should just leave this blank -- if the pem file missing, let's throw an exception in the handler.